lib/plan-builder-generated.js

```diff
@@ -6942,6 +6991,154 @@ bindParam(...args) {
 
     }
 }
+/**
+  * PatchBuilder objects have methods.
+  * @namespace planBuilder.PatchBuilder
+  * @since 3.4.0
+  */
+class PlanPatchBuilder extends types.ServerType {
+
```

Comment:
Oh hey - could `patchArgs` just be an attribute of this class instead? I think that would solve the thread safety issue - each thread would instantiate its own `PlanPatchBuilder` and therefore get its own `patchArgs` array.

Reply:
Tried that earlier but this or any other class in PlanBuilder do not allow variables. (If i try to add let patchArgs = [] on line 7000 it throws a syntax error). I don't think thread safety should be an issue here. In the test file, we are initializing PlanBuilder just once on line 18 and all the tests are using it. We can consider the tests as different threads here. They are accessing the same class but each have their own patchArgs and the queries are created accordingly.